src/custom-tokens/WETH/WETHNativeConverter.sol

```diff
@@ -124,7 +122,7 @@ contract WETHNativeConverter is NativeConverter {
         );
 
         // Emit the event.
-        emit MigrationStarted(msg.sender, amountOfCustomToken, amount);
+        emit MigrationStarted(amountOfCustomToken, amount);
```

Comment:
there are a few other emits that also have msg.sender, where it might be removed:
```txt
VaultBridgeToken.sol:983:        emit DonatedAsYield(msg.sender, assets);
VaultBridgeToken.sol:1070:        emit DonatedForCompletingMigration(msg.sender, assets);
```

Reply:
Those will be left as is.